ante/evm/05_signature_verification.go

```diff
@@ -68,11 +68,19 @@ func SignatureVerification(
 	allowUnprotectedTxs bool,
 ) error {
 	ethTx := msg.AsTransaction()
+	ethCfg := evmtypes.GetEthChainConfig()
```

Comment:
The EIP-155 chain ID is also being stored in the keeper [here](https://github.com/cosmos/evm/blob/89894286132d0883ce6b258288bc99594106d50f/proto/cosmos/evm/vm/v1/evm.proto#L211) which we have access to.

Maybe we can't make a keeper call in every place we need it though? I think we should remove it and use the config everywhere.

Reply:
Fair enough, I'll see what I can do for that. Fwiw, I copied the `AnteHandle` function above with the cfg getter but I could just pass it in from there instead.

Reply:
Removed in https://github.com/cosmos/evm/pull/138/commits/dba4cbce8873c5997d76f11e28d4cd62219aa104